packages/opencensus-core/src/common/console-logger.ts

```diff
@@ -27,9 +27,6 @@ export class ConsoleLogger implements types.Logger {
   static LEVELS = ['silent', 'error', 'warn', 'info', 'debug', 'silly'];
   level: string;
 
-
-  // TODO: reevaluate options to accept numbers as a parameter
```

Comment:
For each TODO that is removed without changes, please document (as a PR comment) why you decided not to do it.

Reply:
Ok, I'll comment those. This one in specific was removed because being able to use numbers as parameters gives more freedom to the end user. Also, passing number as parameters is being currently used across the implementation.

Reply:
Makes sense, thanks.